docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "IncorrectEventABIInLibraries",
```

Comment:
Ah actually this also used "ABI" which I don't think is very relevant here. Perhaps `IncorrectEventSignatureInLibraries`?